app/api/groups/update_group.go

```diff
@@ -17,9 +17,15 @@ package groups
 //     "$ref": "#/definitions/groupUpdateInput"
 // responses:
 //   "200":
-//     "$ref": "#/responses/updated"
-//   default:
-//     "$ref": "#/responses/error"
+//     "$ref": "#/responses/updatedResponse"
+//   "400":
+//     "$ref": "#/responses/badRequestPOSTPUTPATCHResponse"
+//   "401":
```

Comment:
Not sure if we ever return 401 somewhere currently

Reply:
I think we do in the auth middleware